app/v1/controllers/requestParsers/validators/validations/ClaimTypeValidation.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators.validations
+
+import v1.models.domain.ClaimType
+import v1.models.errors.{ClaimTypeFormatError, MtdError}
+
+import scala.util.{Failure, Success, Try}
+
+object ClaimTypeValidation {
+  def validate(claimType: String): List[MtdError] = {
```

Comment:
Looking at how enums are handled in this repo, this could be shortened to
```
def validate(claimType: String): List[MtdError] =
    if (ClaimType.parser.isDefinedAt(claimType)) NoValidationErrors else List(ClaimTypeFormatError)
```